### PR TITLE
Add react class component example

### LINKED ISSUE #1: How to implement in class based component in older react version
Hi Team,
I am facing an issue while implementing the flagsmith in older react with class based component. As you know that we can't use the hooks and javascript code is too lengty while implemeting the code. Can you share the piece of class based component code?

### PR DESCRIPTION
**Pull request by novakzaballa**
Adds a React example using Class components instead of functional components.

### CODE REVIEW

File: react-class-component/src/App.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    if (prevState.identity !== this.state.identity) {
+      this.updateFlags()
+    }
+  }
+
+  updateFlags() {
+    try {
+      const fontSizeEnabled =  flagsmith.hasFeature('font_size');
+      const fontSize =  flagsmith.getValue('font_size');
+      const exampleTrait =  flagsmith.getTrait('example_trait');
```
Comment (khvn26) [inline react-class-component/src/App.js]: nit: do double spaces after `=` mean something?

File: react-class-component/src/App.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  }
+
+  render() {
+    const { fontSizeEnabled, fontSize, trait, identity } = this.state
+
+    return (
+      <div>
+        <h1>Flagsmith Example</h1>
+        <div className='App'>
+          font_size: {fontSize} example_trait: {trait}
```
Comment (khvn26) [inline react-class-component/src/App.js]: Should we display `fontSizeEnabled` somehow? Also would be cool to actually render the content with different font size.

**Review by khvn26 — APPROVED** [review]

File: react-class-component/src/App.js (hunk 1 above)
Comment (novakzaballa) [reply react-class-component/src/App.js]: Corrected.

File: react-class-component/src/App.js (hunk 2 above)
Comment (novakzaballa) [reply react-class-component/src/App.js]: Good idea, I'll implement it before merging it.

**Pull request merged**